modules/manage/pages/iceberg/access-iceberg-topics.adoc

```diff
@@ -0,0 +1,103 @@
+
+* `object_storage` (default): Write catalog files to the same object storage bucket as the data files. Use the object storage URL with an Iceberg client to access the catalog and data files for your Redpanda Iceberg tables.
+* `rest`: Connect to and update an Iceberg catalog using a REST API. See the https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml[Iceberg REST Catalog API specification].
+
+The quickest way to access the data in Iceberg-enabled topics is to use the `object_storage` catalog type, and point your query engine to the table metadata path in the bucket or container. For production use cases, it is recommended to use a REST catalog integration to more easily manage governance, and ensure discoverability and consistency across multiple engines or tools. In either case, you use the catalog to load, query, or refresh the Iceberg data as you produce to the Redpanda topic. For specific guidance on adding the Iceberg tables to your data warehouse or lakehouse using the catalog, refer to the official documentation of your query engine or Iceberg-compatible tool. 
+
+Once you have xref:manage:iceberg/topic-iceberg-integration.adoc#enable-iceberg-integration[enabled the Iceberg integration] for a topic and selected a catalog type, you cannot switch to another catalog type.
+
+== File system-based catalog (`object_storage`)
+
+If you are using the `object_storage` catalog type, you must also set up the catalog integration in your processing engine using the location of the metadata in the bucket or container.
+
+For example, you can configure Spark to use a file system-based catalog with at least the following properties, is using AWS S3 for object storage:
+
+.Example Spark configuration for `object_storage` catalog type
+[,spark]
+----
+spark.sql.catalog.streaming.type = hadoop
+spark.sql.catalog.streaming.warehouse = s3a://<bucket-name>/path/to/redpanda-iceberg-table
+----
+
+Depending on your processing engine, you may need to specify the location for the metadata to xref:manage:iceberg/query-iceberg-tables.adoc[query the Iceberg table]. You may also need to create a new table in your data warehouse or lakehouse for the Iceberg data.
+
+=== Specify metadata location
+
+The config_ref:iceberg_catalog_base_location,true,properties/cluster-properties[] property stores the base path for the file-system based catalog if using the `object_storage` catalog type. The default value is `redpanda-iceberg-catalog`. 
+
+CAUTION: Do not change the `iceberg_catalog_base_location` value after you have enabled Iceberg integration for a topic.
+
+If your engine needs the full JSON metadata path, use the following:
+
+```
+redpanda-iceberg-catalog/metadata/redpanda/<topic-name>/v<version-number>.metadata.json
+```
+
+This provides read access to a single snapshot (denoted by `version-number`) of a table. 
+
+NOTE: Redpanda automatically removes expired snapshots on a periodic basis. 
+
+
+== REST catalog integration
```

Comment:
I'd move this higher on this page. Document first the production grade solution, then the "lesser" ones so the user doesn't get started on the "wrong" path.